subsys/net/lib/zstream/zstream_tls.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <sys/types.h>
+#include <stdbool.h>
+#include <errno.h>
+
+#if !defined(CONFIG_MBEDTLS_CFG_FILE)
+#include "mbedtls/config.h"
+#else
+#include CONFIG_MBEDTLS_CFG_FILE
+#endif /* CONFIG_MBEDTLS_CFG_FILE */
+#include "mbedtls/platform.h"
+
+#include <mbedtls/net.h>
+#include <mbedtls/ssl.h>
+#include <mbedtls/x509_crt.h>
+#include <mbedtls/pk.h>
+#include <mbedtls/entropy.h>
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/debug.h>
+
+#ifdef __ZEPHYR__
+#include <net/socket.h>
+#else /* __ZEPHYR__ */
+#include <unistd.h>
+#include <sys/socket.h>
+#define NET_ERR(msg, ...) printf(msg "\n", ##__VA_ARGS__)
+
+#endif /* __ZEPHYR__ */
+
+#include <net/zstream_tls.h>
+
+#include "../samples/net/echo_server/src/test_certs.h"
```

Comment:
@mbolivar : Thanks for pointing at this leftover line with test_certs.h, removing it right away.